CMakeLists.txt

```diff
@@ -13,5 +13,8 @@ ament_package(
   CONFIG_EXTRAS "python_cmake_module-extras.cmake"
 )
 
+include(cmake/Modules/CheckCrossCompilingSoabi.cmake)
+CheckCrossCompilingSoabi()
```

Comment:
Fail early instead of on usage of this library (which is after compiling the rmw middleware).